admin-ui/plugins/auth-server/redux/sagas/AcrsSaga.js

```diff
@@ -30,12 +36,17 @@ export function* getCurrentAcrs() {
 }
 
 export function* editAcrs({ payload }) {
+  const audit = yield* initAudit()
   try {
+    addAdditionalData(audit, UPDATE, BASIC, { message: payload?.data?.userMessage })
     const api = yield* newFunction()
     const data = yield call(api.updateAcrsConfig, payload.data)
+    yield put(updateToast(true, 'success'))
     yield put(editAcrsResponse({ data }))
+    yield call(postUserAction, audit)
   } catch (e) {
```

Comment:
Failing to log an audit event for a failed operation is bad because it violates the principle of complete auditing by only creating a log entry on the success path of a transaction and omitting it from the failure path, which creates a blind spot in the system's activity log.